vyper/venom/analysis/liveness.py

```diff
@@ -55,6 +55,7 @@ def _calculate_out_vars(self, bb: IRBasicBlock) -> bool:
         Returns True if out_vars changed
         """
         out_vars = bb.out_vars.copy()
```

Comment:
we can skip this copy now
```suggestion
        out_vars = bb.out_vars
```